tests/UI/campaigns/functional/FO/09_productPage/03_writeReview.js

```diff
@@ -68,11 +68,46 @@ describe('FO - product page : Write a review', async () => {
     await helper.closeBrowserContext(browserContext);
   });
 
+  describe(`Install '${moduleInformation.name}' module`, async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to module catalog page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToModuleCatalogPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.modulesParentLink,
+        dashboardPage.moduleCatalogueLink,
+      );
+
+      await moduleCatalogPage.closeSfToolBar(page);
+
+      const pageTitle = await moduleCatalogPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(moduleCatalogPage.pageTitle);
+    });
+
+    it('should install the module', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'installModule', baseContext);
+
+      try {
+        await moduleCatalogPage.searchModule(page, moduleInformation.tag, moduleInformation.name);
+        const successfulMessage = await moduleCatalogPage.installModule(page, moduleInformation.name);
+        await expect(successfulMessage).to.contain(
+          moduleCatalogPage.installMessageSuccessful(moduleInformation.tag),
+        );
+      } catch (e) {
+        // Module already installed, continue test
```

Comment:
Are you sure ? Maybe we should have the option to say
"yes if module is already installed, I don't care, skip step, continue"
or
"no module was not supposed to be installed, better consider something is wrong"